src/js/components/TextInput/TextInput.js

```diff
@@ -363,6 +364,7 @@ const TextInput = forwardRef(
               if (suggestions && suggestions.length > 0) {
                 announce(messages.suggestionsExist);
                 setShowDrop(true);
+                if (onSuggestionsOpen) onSuggestionsOpen();
```

Comment:
I think this call, and the call on line 366 are redundant. 
what about leveraging the `openDrop` function instead of those two?
```suggestion
                openDrop();
```
`openDrop();` will take care of `setShowDrop(true);`  and calling onSuggestionsOpen.
I noticed no one else is calling this function and that may be the right place to use it. 
thoughts?

Reply:
for quick reference https://github.com/grommet/grommet/blob/f4611c750e297e560d35a6403f2eae41a20a4ad7/src/js/components/TextInput/TextInput.js#L166

Reply:
Good call.